README.md

```diff
@@ -42,9 +45,26 @@
 }
 ```
 
-* Invite the bot to your own server, replacing `<YOUR BOT ID>` with... your bot ID: `https://discordapp.com/api/oauth2/authorize?client_id=<YOUR BOT ID>&permissions=286280784&scope=bot`
+* Invite the bot to your own server, replacing `<YOUR CLIENT ID>` with... your bot ID: `https://discordapp.com/api/oauth2/authorize?client_id=<YOUR CLIENT ID>&permissions=286280784&scope=bot`
```

Comment:
Keep it as BOT not CLIENT because that creates confusion.

Reply:
the reason for this change is the ID required is titled Client ID for the application, i'll revert it back to BOT